composer.json

```diff
@@ -92,6 +92,8 @@
         "ext-soap": "*",
         "ext-openssl": "*",
         "ext-zip": "*",
+        "ext-imap": "*",
```

Comment:
A gdy ktoś nie używa skryptów backendowych z modułem php-imap?
Nie wiem, czy to dobre, żeby wymagać tego.
Z tego, co pamiętam próbowałem już kiedyś tym sposobem narzucić więcej sensownych wymagań - efekt tego był taki, że w niektórych dystrybucjach krzyczało, że moduł php jest wymagany, mimo że był zainstalowany.
Z ext-iconv możemy spróbować, ale najpierw zrobię testy u siebie na kilku różnych środowiskach.
Dam znać jeszcze w kwestii narzucenia wymogu obydwu modułów PHP. Pewnie dzisiaj.

Reply:
Dałem commitem ffc3476e5 wymóg ext-iconv. ext-iconv w paru instalacjach nie jest przeze mnie używany i tam robiło to problem.

Reply:
A jak ktoś nie korzysta z tej części systemu co wykorzystuje wymaganie obecne już w composerze xml?;)

Ja tylko chce konsekwencji w tym...

Reply:
Jak się objawiał problem?

Reply:
Brakiem dostępności modułu imap.
XML jednak jest częściej używany.
Moim zdaniem chcesz to widzieć czarno-biało (albo inaczej 1 vs 0), a tak w rzeczywistości rzadko się da.

Reply:
Komputery są zero jedynkowe ;)

Wracając do problemu z wymaganiami - jak się on objawiał?